source/_integrations/wake_on_lan.markdown

```diff
@@ -133,3 +133,30 @@ switch:
 shell_command:
   turn_off_TARGET: "ssh hass@TARGET sudo pm-suspend"
 ```
+
+## Helper Button with Automation
+
+A switch defined with the `wake_on_lan` platform will render in the UI with both 'on' and 'off' clickable actions. If you don't intend to use the `turn_off` functionality then using a virtual button & automation will look cleaner and less confusing. It will only have one action.
```

Comment:
```suggestion
A switch defined with the `wake_on_lan` platform will render in the UI with both 'on' and 'off' clickable actions. If you don't intend to use the `turn_off` functionality, then using a virtual button & automation will look cleaner and less confusing. It will only have one action.
```